scripts/init_sqlite_wasm.dart

```diff
@@ -1,8 +1,8 @@
-/// Downloads sqlite3.wasm and copies it to all demo folders
+/// Downloads sqlite3.wasm into the assets directory for tests
```

Comment:
Do we still need this script? Can't the new tool download this for us?

Reply:
The problem with using the new script is that it is run multiple times for the demos and if we add it to download to the assets it will download multiple times for each demo that has a web folder. If you look at the `melos prepare` command thats how it currently functions. I could rework it to add an additional argument that downloads it to assets just once.

The other consideration is that users will be running the script and they don't need that functionality for their applications.

Reply:
File has been removed in favor of an output directory argument.